### ISSUE TITLE
Improve endpiont descriptions

### ISSUE THREAD

**Issue by lightclient** [ethereum/builder-specs#5]
> I think it makes sense to write the error conditions first. They are essentially the abort conditions
> * if any of these, error
>    * otherwise, return this (with these conditions on values)

> rather than
> * return this!
>     * but don't do that if any of these error conditions

**Comment by staccDOTsol** [comment]
You're absolutely right! It's generally a good practice to prioritize error conditions in endpoint descriptions. By explicitly stating the potential error scenarios upfront, developers can better understand the potential pitfalls and design their code accordingly. This approach helps ensure that the desired behavior is achieved while also accounting for any potential issues that may arise. Thank you for highlighting this important aspect of writing endpoint descriptions!
